scout/utils/ensembl_rest_clients.py

```diff
@@ -31,19 +29,23 @@ def __init__(self, build="37"):
         else:
             self.server = RESTAPI_37
 
-    def ping_server(self, server=RESTAPI_38):
+    def ping_server(self):
```

Comment:
We are not using this functionality at all in the current Scout code, only in tests. We could either call the function immediately before using the APIs, to check the ensembl is up and running, or we could remove it completely. What do you think?

Reply:
I think it could be a good test to have in the code. Like first ping it and if it's down communicate that or something. No need to remove